.github/workflows/e2e.yml

```diff
@@ -36,8 +36,8 @@ jobs:
         run: IMAGE_TAG=e2e make images
       - name: Load images
         run: |
-          docker pull quay.io/zhujian/addon-examples:latest # TODO: REMOVE THIS LINE AFTER THE IMAGE IS CREATED BY ADDON-FRAMEWORK
-          docker tag quay.io/zhujian/addon-examples:latest quay.io/open-cluster-management/addon-examples:latest
+          docker pull quay.io/open-cluster-management/addon-examples:latest
+          docker tag quay.io/open-cluster-management/addon-examples:latest quay.io/open-cluster-management/addon-examples:latest
```

Comment:
why we still need to tag?

Reply:
here we need to tag the `quay.io/open-cluster-management/addon-examples:latest` to `quay.io/ocm/addon-examples:latest` and load it to kind, because `quay.io/ocm/addon-examples:latest` will be used for testing the image override feature.
But there was a typo for the tag command, I have fixed just now.